pkg/domain/infra/abi/containers.go

```diff
@@ -540,7 +542,7 @@ func (ic *ContainerEngine) ContainerRestore(ctx context.Context, namesOrIds []st
 
 	switch {
 	case options.Import != "":
-		cons, err = checkpoint.CRImportCheckpoint(ctx, ic.Libpod, options.Import, options.Name)
+		cons, err = checkpoint.CRImportCheckpoint(ctx, ic.Libpod, options.Import, options.Name, options.IgnoreVolumes)
```

Comment:
In exportCheckpoint() you changed the function to only have the `options` parameter instead of one parameter for each used option. I guess you could to the same here.

Reply:
Thank you for suggesting that. I have applied this change in an additional commit.